pkg/sdk/tag_association_validations.go

```diff
@@ -71,6 +73,10 @@ var (
 	TagAssociationAllowedObjectTypesString = make([]string, len(TagAssociationAllowedObjectTypes))
 )
 
+func isTagAssociationAllowedObjectTypes(o ObjectType) bool {
```

Comment:
nit: rename to `canBeTagged` or `canBeAssociatedWithTag` or something else

Reply:
Done.